pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -89,6 +90,35 @@ func (pl *VolumeBinding) Name() string {
 	return Name
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEvent {
+	events := []framework.ClusterEvent{
+		// Pods may fail because the storage class is not configured correctly.
+		// After the storage class is fixed (e.g. allowedTopologies,
+		// volumeBindingMode), the pods may become schedulable.
+		{Resource: framework.StorageClass, ActionType: framework.Add | framework.Update},
+		// We binds PVCs with PVs, so any changes may make the pods schedulable.
+		{Resource: framework.PersistentVolumeClaim, ActionType: framework.Add | framework.Update},
+		{Resource: framework.PersistentVolume, ActionType: framework.Add | framework.Update},
+		// Pods may fail to find available PVs because the node labels does not
+		// fix in the storage class's allowed topoligies or PV's node affinity
+		// criteria. Changes to node may make pods schedulable.
```

Comment:
Does this read better?

```
// Pods may fail to find available PVs because the node labels do not
// match the storage class's allowed topologies or PV's node affinity.
// A new or updated node may make pods schedulable.
```

Reply:
sounds better!